src/gluonts/nursery/autogluon_tabular.py

```diff
@@ -0,0 +1,118 @@
+from gluonts.model.predictor import Predictor
```

Comment:
The header for both files is missing; you can add it by running

```
.devtools/license fix src test
```

and then committing and pushing the resulting changes